tests/conftest.py

```diff
@@ -64,3 +66,71 @@ def ensure_default_subclasses(db):
         obj2.set_current_language("lt")
         obj2.title = "Duomenų publikavimo paslauga"
         obj2.save()
+
+
+# TODO: remove this after the pipeline starts running migrations before testing
+@pytest.fixture(autouse=True)
+def ensure_needed_concepts_exist():
+    schema, _ = ConceptSchema.objects.get_or_create(
+        uri="http://publications.europa.eu/resource/authority/distribution-status"
+    )
+
+    concepts_data = [
+        {
+            "uri": "http://publications.europa.eu/resource/authority/distribution-status/COMPLETED",
+            "code": "COMPLETED",
+            "valid_since": date(2015, 10, 23),
+            "translations": {
+                "en": {
+                    "label": "Completed",
+                    "description": "This distribution is considered to be complete, it holds all information that is intended.",
+                },
+                "lt": {
+                    "label": "Įgyvendintas – veikiantis",
+                    "description": "Ši distribucija laikoma įgyvendinta – veikiančia, joje yra visa reikiama informacija.",
+                },
+            },
+        },
+        {
+            "uri": "http://publications.europa.eu/resource/authority/distribution-status/DEVELOP",
+            "code": "DEVELOP",
+            "valid_since": date(2015, 10, 23),
+            "translations": {
+                "en": {
+                    "label": "Under development",
+                    "description": "This distribution is currently being assembled. It may be in an incomplete or faulty state.",
+                },
+                "lt": {
+                    "label": "Kuriamas",
+                    "description": "Ši distribucija yra kuriama. Ji gali būti nebaigta arba klaidinga.",
+                },
+            },
+        },
+        {
+            "code": "PLANNED",
+            "valid_since": date(2019, 1, 1),
+            "translations": {
+                "en": {
+                    "label": "Development planned",
+                    "description": "The development of this distribution is planned.",
+                },
+                "lt": {"label": "Kūrimas suplanuotas", "description": "Šios distribucijos kūrimas suplanuotas."},
+            },
+        },
+    ]
+
+    for data in concepts_data:
+        concept, created = Concept.objects.get_or_create(
+            code=data["code"],
+            defaults={
+                "uri": data.get("uri"),
+                "valid_since": data["valid_since"],
+            },
+        )
+        if created:
+            for lang, fields in data["translations"].items():
+                concept.set_current_language(lang)
+                concept.label = fields["label"]
+                concept.description = fields["description"]
+            concept.save()
+        concept.concept_schemas.add(schema)
```

Comment:
I suggest doing just `create` instead of `get or create` (if possible), because of two reasons:

1. performance increase - as this fixtures `scope` currently is `function` (default), its executed before each test run.
2. once migrations will be enabled (during test run) with only create logic  - it will fail instantly and won't be left as a dead code.